client/my-sites/customer-home/style.scss

```diff
@@ -207,26 +199,7 @@
 	&__grow-earn {
 		padding-bottom: 12px;
 	}
-	&__go-mobile.is-single-store {
-		display: flex;
-		padding-bottom: 1rem;
-
-		.customer-home__card-subheader,
-		.get-apps__app-badge img {
-			margin: 0;
-		}
-
-		.get-apps__app-badge {
-			width: auto;
-		}
 
-		.customer-home__card-mobile {
-			margin-left: auto;
-			align-self: center;
-			flex-shrink: 0;
-			padding-left: 1rem;
-		}
```

Comment:
These styles seem to be missing as well in the new .scss file

Reply:
Fixed in #3c48066

Reply:
Thanks @mmtr I probably missed a few from the &__ sass shorthand. 

As an aside, for folks reading we should avoid this since while it's less typing it's more difficult to read and we can't perform a basic string search to find related code/usages.